config/image.go

```diff
@@ -55,6 +55,7 @@ type ImageConfig struct {
 	// Pull Pull an image instead of building it. The value may be one of:
 	// * ``once`` - only pull if the image:tag does not exist
 	// * ``always`` - always pull the image
+	// * ``local`` - don't pull or build the image. Use one that is already present locally
```

Comment:
I think the term `never` would be more appropriate here. It fits better with `once`, and `always`. The description sounds good, and shouldn't need to change.

Reply:
You're right, `never` sounds indeed better.